### PR TITLE
Add checksum for py-urllib3 1.26.12

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-urllib3/package.py [hunk 1]
```diff
@@ -28,6 +29,7 @@ class PyUrllib3(PythonPackage):
 
     depends_on("python@2.7:2.8,3.4:", when="@:1.25", type=("build", "run"))
     depends_on("python@2.7:2.8,3.5:", when="@1.26.6", type=("build", "run"))
+    depends_on("python@2.7:2.8,3.6:", when="@1.26.12:", type=("build", "run"))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-urllib3/package.py]:
```suggestion
    depends_on("python@2.7:2.8,3.6:3", when="@1.26.12:", type=("build", "run"))
```

File: var/spack/repos/builtin/packages/py-urllib3/package.py [hunk 2]
```diff
@@ -28,6 +29,7 @@ class PyUrllib3(PythonPackage):
...
 
     depends_on("py-setuptools", type="build")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-urllib3/package.py:35]: `+secure` has a new dep on `py-urllib3-secure-extra`

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-urllib3/package.py (hunk 2 above)
Comment (iarspider) [reply var/spack/repos/builtin/packages/py-urllib3/package.py:35]: I think we should even remove the `+secure` variant: https://github.com/urllib3/urllib3/issues/2680

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-urllib3/package.py:35]: Looks like it's only needed by `py-azure-cli`. You could see if that's still true in the latest release. Or you could copy-n-paste the `+secure` deps from this package to `py-azure-cli`. Or leave it as is and update the dep.

**Review by adamjstewart — APPROVED** [review]

**Comment by alalazo** [comment]
@spackbot run pipeline

**Pull request merged**